Formula/hyperkit.rb

```diff
@@ -0,0 +1,39 @@
+class Hyperkit < Formula
+  desc "Toolkit for embedding hypervisor capabilities in your application"
+  homepage "https://github.com/moby/hyperkit"
+  url "https://github.com/moby/hyperkit/archive/v0.20180123.tar.gz"
+  sha256 "382933118da3835056203d3d05923b554f36cc41a555a821516e11ccb7d16bf3"
+
+  head do
+    url "https://github.com/hishamhm/htop.git"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
+  def install
+    system "make"
+    bin.install "build/hyperkit"
+  end
+
+  def caveats; <<~EOS
+    To enable qcow support in the block backend an OCaml OPAM development
```

Comment:
This formula should be built with qcow support rather than having a caveat.

Reply:
Ok I though about it, thanks for your feedback

Reply:
do you have other feedbacks, or is it ok to merge this? @ilovezfs